### ISSUE TITLE
mypy raises error on writing msgpack to binary file 

### ISSUE THREAD

**Issue by al-zatv** [Fatal1ty/mashumaro#63]
Line of code which save msgpack to binary file, cause mypy to raise typing error.  How can I fix typing here? Save/load works fine, by the way.

```
@attr.s(auto_attribs=True)
class Phrase(DataClassMessagePackMixin):
    some_data: str
....

with open(base_data_path / "my_data", "wb") as f:
        f.write(phrase.to_msgpack())

```
Here, mypy raises typing error: 
  error: Argument 1 to "write" of "IO" has incompatible type "Union[str, bytes, bytearray]"; expected "bytes"

# mypy --version
mypy 0.930

**Comment by Fatal1ty** [comment]
Hello @al-zatv 

At first glance it seems like `to_msgpack` method in `DataClassMessagePackMixin` should return `bytes`. I will see if we can change it from `Union[str, bytes, bytearray]` without problems. As a workaround you can use [cast](https://docs.python.org/3/library/typing.html#typing.cast):
```python
f.write(cast(bytes, phrase.to_msgpack()))
```

**Comment by Fatal1ty** [comment]
Fixed in 2.10.
